src/Microsoft.ML.Core/Data/ModelHeader.cs

```diff
@@ -10,7 +10,8 @@
 
 namespace Microsoft.ML.Model
 {
-    [StructLayout(LayoutKind.Explicit, Size = ModelHeader.Size)]
+    [StructLayout(LayoutKind.Explicit, Size = Size)]
+    [BestFriend]
```

Comment:
Do we want to sort attributes?

Reply:
Didin't know about this best practice, thank you for letting me know, it makes sense :)

---
In reply to: [254933949](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254933949) [](ancestors = 254933949)